tests/test_metadata.py

```diff
@@ -143,3 +149,20 @@ def test_submit_metadata_manifest(
                     hide_blanks=hide_blanks,
                 )
                 assert mock_manifest_id == "mock manifest id"
+
+    def test_submit_filebased_manifest(self, helpers):
+        meta_data_model = metadata_model(helpers, "class_label")
+
+        manifest_path = helpers.get_data_path(
+            "mock_manifests/filepath_submission_test_manifest.csv"
+        )
+
+        with does_not_raise():
```

Comment:
Could you explain the rationale behind this?

Reply:
Yeah for sure, it was in the `pytest` documentation for testing that an exception is raised or not raised under certain conditions. For this test the important thing was to make sure the `submit_metadata_manifest` was able to complete without raising an exception. I added the `with does_not_raise` to try and be more explicit about that. I wanted to avoid having a test that just called the method and did nothing else. I guess with the assertion with `with` statement kind of unnecessary but that was added later and isn't the main point of the test and I wasn't sure what the best combination was in this case.
@thomasyu888

Reply:
Thanks!  I also could've read the pytest documentation: https://docs.pytest.org/en/6.2.x/example/parametrize.html#parametrizing-conditional-raising. 😅

Reply:
@thomasyu888 Yeah you're welcome!